hack/verify-bash-completion.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${STI_ROOT}/hack/common.sh"
+
+pushd "${STI_ROOT}" >/dev/null
+
+mv contrib/bash/s2i contrib/bash/s2i-proposed
+hack/update-generated-completions.sh
+
+ret=0
+diff -Naupr contrib/bash/s2i contrib/bash/s2i-proposed || ret=$?
```

Comment:
If you put the `diff ...` as the `if` condition, we don't need to manipulate the return code.

```bash
if diff ...; then
   ...
else
   ...
```